ui/selectors/accounts.ts

```diff
@@ -12,3 +12,10 @@ export function isSelectedInternalAccountEth(state: AccountsState) {
 
   return Boolean(account && (account.type === Eoa || account.type === Erc4337));
 }
+
+export function isSelectedInternalAccountBtc(state: AccountsState) {
```

Comment:
Should not be part of this PR

Reply:
Removed in daa5e47fb99450b9abc38f37f67a94349a733a75